engine/src/test/java/com/sandboni/core/engine/finder/bcel/BcelFinderTest.java

```diff
@@ -643,4 +642,45 @@ public void testCucumberRunnerDetection() {
                 LinkType.CUCUMBER_RUNNER);
         testTestClassVisitor(expectedLink);
     }
+
+    @Test
+    public void testInterfaceWithLambdas() {
+        Link interfaceToImplementingClass = newLink(new Vertex.Builder("java.util.function.Function", String.format("apply(%s.BlotterInput)", LAMBDA_PACKAGE_NAME)).build(),
```

Comment:
Some of the Strings can be defined as constants.

Reply:
I agree that constants should be used when we are repeating the string value. In this scenario, however, makes clearer to have the actual string to see what values are expected. If you don't know the code and are reading the test, is easier to understand what values should the Link contains instead of the overhead of check the constant value. The whole test class is defined in this way.